webblog/frontend/src/components/blog/Form.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from "react";
+import {connect} from "react-redux";
+import PropTypes from "prop-types";
+import {addBlog} from "../../actions/blogList";
+
+export class Form extends Component {
+    static propTypes = {
```

Comment:
What is the reason of using prop-types ?

Reply:
it is used to require a function addBlog or else it'll give an error before loading the component, which is better than not loading the component.